.travis.yml

```diff
@@ -1,3 +1,8 @@
+# Use the fully virtualised infra not for sudo, but since their container
```

Comment:
s/, but//

Reply:
That changes the meaning to the opposite of the what it's intending to say.

To explain in another way - Travis has two trusty based stacks:
* container (`sudo: false`): no-sudo, less RAM
* fully virtualised (`sudo: required`): has sudo, more RAM, few other differences

Much of the time people use the latter because of the need to use `sudo` commands rather than because of the difference in machine specifications, which is why Travis use the `sudo: {false,required}` setting to control which environment is used. 

However in this case, the need for sudo commands is not why we're setting `sudo: required`, it's because we want the higher spec fully virtualised environment.

I'm open to suggestions to another way of working that makes that clearer?

Reply:
No no, I was just confused as to why travis configures it this way. Proceed. 😄